$OpenBSD$

set secure_delete PRAGMA on by default, instead of using internal sqlite copy
https://bugzilla.mozilla.org/show_bug.cgi?id=546162

Patch taken from www/firefox-esr
--- storage/src/mozStorageConnection.cpp.orig	Tue Sep  1 08:53:38 2015
+++ storage/src/mozStorageConnection.cpp	Wed Sep  9 15:58:57 2015
@@ -713,6 +713,13 @@ Connection::initializeInternal(nsIFile* aDatabaseFile)
     return convertResultCode(srv);
   }
 
+  srv = ::sqlite3_exec(mDBConn, "PRAGMA secure_delete = ON", NULL, NULL, NULL);
+  if (srv != SQLITE_OK) {
+    ::sqlite3_close(mDBConn);
+    mDBConn = nullptr;
+    return convertResultCode(srv);
+  }
+
   // Set the synchronous PRAGMA, according to the preference.
   switch (Service::getSynchronousPref()) {
     case 2:
